src/types.ts

```diff
@@ -462,6 +462,14 @@ export type TigrisSchema<T> = {
 	};
 };
 
+export type TigrisTopicSchema<T> = {
```

Comment:
could you please add test in `tigris.schema.spec.ts` to exercise code path using TigrisTopicSchema .

Reply:
The `createOrUpdateTopic` test exercises it.

Reply:
ah ok 👍